File: sem 3/python labbbb/Day 4 7th oct/40_program3.py
s=input("Enter a string ")
low=0
high=len(s)-1
fl=0
while low<=high:
    if s[low]!=s[high]:
       fl=1
       break
    low=low+1
    high=high-1

if fl==0:
    print(s,"is palindrome")
else: 
    print(s,"is not palindrome")

File: sem 2/computer lab 23rd june/40_program3.c
#include<stdio.h>
int fact(int n)
{
if(n==0 ||n==1)//base case
    return 1;
return n*fact(n-1);
}
int main()
{
    int n;
    printf("Enter the limit\n");
    scanf("%d",&n);//taking input
    double sum=0.0;
    for(int i=1;i<=n;i++)
    {
     sum=sum+((1.0)*(2*i-1)*(i+2)/fact(i));
    }
    printf("Sum is %0.2lf \n",sum);
    return 0;
}